### PR TITLE
"Fix jinja render bug if 'portal_net' is undefined"

### PR DESCRIPTION
**Pull request by litian54**
Define a default value of 'portal_net'.

### CODE REVIEW

**Comment by thockin** [comment]
LGTM

File: cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest [hunk 1]
```diff
@@ -35,6 +36,7 @@
 
 {% set etcd_servers = "--etcd_servers=http://127.0.0.1:4001" -%}
 
+{%  set portal_net = "" -%}
```
Comment (zmerlynn) [inline cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest]: Extra whitespace here between "%" and "set".

**Comment by litian54** [comment]
I've fixed the extra whitespace.

**Comment by zmerlynn** [comment]
Sorry for the hassle, but can you squash now? There's some funky going on with your branch, it's showing 9 commits.

**Pull request closed without merging**